ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/SqlLedger.scala

```diff
@@ -233,7 +243,7 @@ private class SqlLedger(
       }
 
       val recordTime = timeProvider.getCurrentTime
-      if (recordTime.isAfter(submitterInfo.maxRecordTime.toInstant)) {
+      val entry = if (recordTime.isAfter(submitterInfo.maxRecordTime.toInstant)) {
```

Comment:
Arguably, the RT < MRT check is trivial. But shouldn't it be a part of the TimeModel, so that all checks are colocated for an easy reference and change?